pkg/api/validation/validation.go

```diff
@@ -358,12 +362,26 @@ func validateNFS(nfs *api.NFSVolumeSource) errs.ValidationErrorList {
 	return allErrs
 }
 
+func validatePersistentVolumeClaimVolumeSource(claimAttachment *api.PersistentVolumeClaimVolumeSource) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+
+	if claimAttachment.ClaimName == "" {
```

Comment:
len(claimAttachment.ClaimName) == 0 seems to be the preferred route i get asked to use